arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,768 @@
+// SPDX-License-Identifier: GPL-2.0+ or X11
+/*
+ * Copyright (C) 2021-2023 Technologic Systems, Inc. dba embeddedTS
+ */
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include "imx6ul.dtsi"
+
+/ {
+	model = "embeddedTS i.MX6UL TS-7180";
+	compatible = "technologic,ts7180", "fsl,imx6ul";
+
+	aliases {
+		ethernet0 = &fec2;
+		ethernet1 = &fec1;
+	};
+
+	chosen {
+		stdout-path = &uart1;
+	};
+
+	memory@80000000 {
+                device_type = "memory";
+                /* Memory size is to be filled in by U-Boot */
+		reg = <0x80000000 0>;
+	};
+
+	led-controller {
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_gpio_leds>;
+		compatible = "gpio-leds";
+
+		led-2 {
```

Comment:
Any reason to make these led-2 3 4 6  and not number from 0-3?

Reply:
These had been following the reference designators on the schematic; not so much needed with the way LEDs are now specified. Will renumber to 0